StripePaymentSheet/StripePaymentSheet/Source/Internal/API Bindings/v1-elements-sessions/ElementsCustomer.swift

```diff
@@ -59,6 +59,8 @@ struct ElementsCustomer: Equatable, Hashable {
                     let paymentMethod = paymentMethodWithLinkDetails.paymentMethod
                     if let linkDetails = paymentMethodWithLinkDetails.linkDetails {
                         paymentMethod.setLinkPaymentDetails(from: linkDetails)
+                    } else {
+                        paymentMethod.isLinkPassthroughMode = paymentMethodWithLinkDetails.isLinkOrigin
```

Comment:
A payment method is only created in Link passthrough if `isLinkOrigin` is true but `linkDetails` is nil.